test/integration/app/prompted_nobuild.js

```diff
@@ -312,20 +313,68 @@ describe('Prompt and no build integration tests for app generator', function ()
         "/persons": {
           "get": {
             "description": "Gets `Person` objects.",
+            "responses": {
+              "200": {
+                "description": "pet response",
+                "schema": {
+                  "$ref": "#/definitions/Person"
+                }
+              }
+            }
```

Comment:
So when I ran the test previously, this Swagger Json was invalid. I added more details to make it valid, but we can simplify the Swagger Json if you would like? Currently, we could get rid of the `/persons` `put` endpoint and the tests will still pass, but I think thats all that can be taken away without further modification.